src/lob-address-elements.js

```diff
@@ -617,19 +637,30 @@ import { parseWebPage } from './form-detection';
          * @param {function} cb - process the response (submit the form or show an error message)
          */
         config.do.verify = function (cb) {
+          config.international = isInternational();
           config.submit = config.strictness === 'passthrough';
           config.confirmed = isConfirmation();
 
-          var payload = {
+          const payload = {
             primary_line: config.elements.primary.val(),
             secondary_line: config.elements.secondary.val(),
             city: config.elements.city.val(),
-            state: config.elements.state.val(),
-            zip_code: config.elements.zip.val()
+            state: config.elements.state.val()
           };
 
-          var xhr = new XMLHttpRequest();
-          var path = config.apis.verify + '?av_elements_origin=' + window.location.href;
+          if (config.international) {
+            const country = config.elements.country.val().toLowerCase();
+
+            payload.country = country.length === 2 ? country : countryCodes[country];
+            payload.postal_code = config.elements.zip.val();
+          } else {
+            payload.zip_code = config.elements.zip.val();
+          }
+
+          const endpoint = config.international ? config.apis.intl_verify : config.apis.us_verify;
+          const path = endpoint + '?av_elements_origin=' + window.location.href;
```

Comment:
Is this the flag for usage? If so, this should be a header.

Reply:
Yes that's the flag, feedback from the [API PR](https://github.com/lob/lob-api/pull/6736) was to avoid non-standard headers when possible. So I switched to using a query param instead

Reply:
I would have preferred a header for something that relates to metadata collection. For now, let's keep the new query parameter private, and we can discuss offline.